src/controllers/expenses.controllers.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+
+const get = async(req, res) => {
+  const { userId, categories, from, to } = req.body;
+
+  const expenses = expensesService.getExpenses({
+    userId, categories, from, to,
+  });
+
+  if (!expenses) {
+    res.sendStatus(404);
+
+    return;
+  }
+  res.send((await expenses).map(expense => expensesService.normalize(expense)));
```

Comment:
probably would be better to move this to service cause you are kinda interacting with a data itself, controllers just receive data from the user sends it to service, receives data from him and sends it to user